developer_docs/foreman_dev_setup.asciidoc

```diff
@@ -132,6 +132,38 @@ All rake tasks are available with the following command:
 bundle exec rake -T
 ....
 
+=== Running tests in Foreman
+Make sure to run tests from the Foreman directory. To run rubocop test, you can use the following commands:
+[source, shell]
+....
+bundle exec rubocop [<path_to_file>]
+bundle exec rubocop -a # Auto-correct offenses (only when it's safe)
+bundle exec rubocop -A # Auto-correct offenses (safe and unsafe)
```

Comment:
In documentation it's usually better to use long options since they're self-describing. So you'd use
```suggestion
bundle exec rubocop --auto-correct # (only when it's safe)
bundle exec rubocop --auto-correct-all # (safe and unsafe)
```

I'd also create a new paragraph for it. The above suggests it's just for testing, but auto correction is its own thing.

Reply:
Thanks for the review! @ekohl